package.json

```diff
@@ -46,13 +49,17 @@
     "mkdirp": "^0.5.1",
     "mocha": "^2.2.4",
     "mocha-jsdom": "^1.0.0",
+    "optimist": "^0.6.1",
     "protractor": "^3.0.0",
+    "q": "^1.4.1",
     "require-dir": "^0.3.0",
+    "selenium-webdriver": "^2.48.2",
```

Comment:
Hmm, I'm not sure about that. `q` doesn't have selenium-webdriver in its `package.json` https://github.com/kriskowal/q

Would you mind trying to disable gulp-sitespeedio and see if you still have dependency issues? I see selenium-webdriver as a dependency there:

```
├─┬ gulp-sitespeedio@0.0.8
│ ├── object-assign@2.0.0
│ ├─┬ sitespeed.io@3.11.1
│ │ ├── async@1.3.0
│ │ ├─┬ browsertime@0.12.0
│ │ │ ├── async@1.5.0
│ │ │ ├─┬ async-node-events@0.0.6 (git+https://github.com/joshuafcole/async-node-events.git#d32411f1f4df70c5dbe68d6a7861b5ed7fb539a3)
│ │ │ │ └── is-async@0.0.2
│ │ │ ├── browsermob-proxy-api@0.3.0
│ │ │ ├── cross-spawn-async@2.0.0
│ │ │ ├── getport@0.1.0
│ │ │ ├── selenium-webdriver@2.47.0
```